cng/big.go

```diff
@@ -2,13 +2,29 @@
 // Licensed under the MIT License.
 package cng
 
+import "math/bits"
+
 // This file does not have build constraints to
 // facilitate using BigInt in Go crypto.
 // Go crypto references BigInt unconditionally,
 // even if it is not finally used.
 
-// A BigInt is the big-endian bytes from a math/big BigInt.
+// A BigInt is the big-endian bytes from a math/big BigInt,
+// which are normalized to remove any leading 0 byte.
 // Windows BCrypt accepts this specific data format.
 // This definition allows us to avoid importing math/big.
 // Conversion between BigInt and *big.Int is in cng/bbig.
 type BigInt []byte
+
+const _S = bits.UintSize / 8 // word size in bytes
```

Comment:
This seems to hard code to 64 bit platforms. Should we use something like `strconv.IntSize`, or a better value, to make the code more portable?